guide/11-performing-network-analyses/guide_to_network_analysis_part1.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Line 1 - please use one of 2 forms of the product name - either "ArcGIS API for Python" or "Python API" to keep in consistent in the guide.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>